_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Book Review: The Intelligent Investor"
+date:   2019-02-24 22:40:05
+categories: jekyll update
+---
+
+### Disclaimer and Basics on Retirement
+
+I am simply an enthusiastic lay person
+who happens to know some math, a fair bit of personal finance,
+and likes reading books on the topic of investing. Please do your own research
+and don't ask me for specific advice about any companies,
+markets, etc.
+
+**I will not help you with individual financial stuff. Please do not ask.** Consult with a fiduciary advisor with any specific question you have and make sure your 401k company is not screwing you over. *Hint*: They are probably screwing you over, watch [The Retirement Gamble](https://www.pbs.org/video/frontline-retirement-gamble/).
+
+
+### The Gist
+
+_The Intelligent Investor_ and _Security Analysis_ is the only investment books I ever needed to read, all the others were further elaboration on Benjamin Graham's points. I decided to do this book review because the principles to preserve wealth, reducing the amount of false positives, and truly invest for the long run (the only investment that exists) are timeless techniques. For the purposes of brevity I will focus primarily on what Benjamin Graham calls a Defensive Investor instead of the Enterprising Investor.
+
+Intelligent investing is not about being the smartest person in the room, it is about following sound principles through to their ends. An intelligent investor is someone who develops temperance, patience, control, is learned, meticulous in their reasoning, defensive in their thinking, and becoming familiar with the major ideas in many fields related to investing. These fields include, but are not limited to: psychology, history, basic statistics, meditation, and human evolution. An intelligent investor has more in common with an endurance athlete than a superstar sports player, in this game consistency can beat talent in many cases.
+
+One of the worst investors of all time is Sir Isaac Newton. I don't think many people would consider him stupid, but he was a stupid investor. 
```

Comment:
This seems interesting. Why is he a stupid investor? I would like to see more about this (if not in this paragraph, in a link or footnote).